bin/dev_command/setup

```diff
@@ -7,12 +7,35 @@ echo '';
 setup() {
 
     mysql() {
-        if [ -z "`${DEV_SUDO} docker volume ls -q -f 'name=dockerdev-mysql-volume'`" ]; then
-            echo 'Create a persistant database volume';
-            ${DEV_SUDO} docker volume create dockerdev-mysql-volume;
+        local mysql_version="$1"
+        local file="${DEV_WORKPATH}/conf/mysql$mysql_version"
+        local endis='';
+        local volume_name="dockerdev-mysql${mysql_version}-volume"
+        local service_name="db${mysql_version}"
+
+        if [ -z "${mysql_version}" ]; then
+            endis='n'
+            mysql_version='5.7'
         fi
 
-        ./${DEV_SELF} setup-mysql db;
+        if [ -z "${endis}" ]; then
```

Comment:
what are you trying to achieve here?

Reply:
This is part of the actual check if the mysql8 config file is there or not.